manifests/w/w4po/ExplorerTabUtility/2.2.0/w4po.ExplorerTabUtility.locale.en-US.yaml

```
@@ -0,0 +1,19 @@
+# Created using wingetcreate 1.9.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.9.0.schema.json
+
+PackageIdentifier: w4po.ExplorerTabUtility
+PackageVersion: 2.2.0
+PackageLocale: en-US
+Publisher: w4po
+PublisherUrl: https://github.com/w4po
+PublisherSupportUrl: https://github.com/w4po/ExplorerTabUtility/issues
+PackageName: ExplorerTabUtility
+PackageUrl: https://github.com/w4po/ExplorerTabUtility
+License: MIT
+ShortDescription: "\U0001F680 Supercharge Windows 11's File Explorer: Auto-convert windows to tabs, duplicate tabs, reopen closed ones, and more!"

```

```suggestion
ShortDescription: "🚀 Supercharge Windows 11's File Explorer: Auto-convert windows to tabs, duplicate tabs, reopen closed ones, and more!"
```

An issue that shouldn't affect the installation.